File: src/Colleen.c
#include <stdio.h>
int f(){return 0;}
/*
  com1
*/
int main(){
	f();
	char *s="#include <stdio.h>%cint f(){return 0;}%c/*%c  com1%c*/%cint main(){%c	f();%c	char *s=%c%s%c;%c/*%c  com2%c*/%c	printf(s,10,10,10,10,10,10,10, 34, s, 34,10,10,10,10,10,10,10);%c	return(0);%c}%c";
/*
  com2
*/
	printf(s,10,10,10,10,10,10,10, 34, s, 34,10,10,10,10,10,10,10);
	return(0);
}

File: bonus/Colleen.py
def f():
    return
#com1
if __name__ == '__main__':
    f()
#   com2
    s = """def f():
    return
#com1
if __name__ == '__main__':
    f()
#   com2
    s = %c%c%c%s%c%c%c
    print(s%%(34,34,34,s,34,34,34))"""
    print(s%(34,34,34,s,34,34,34))
